src/components/sidebar/connection/ConnectionListItem.vue

```diff
@@ -9,8 +9,18 @@
     >
       <span :class="`connection-label connection-label-color-${labelColor}`"></span>
       <div class="connection-title flex-col expand">
-        <span class="title">{{label}}</span>
-        <span class="subtitle">{{subtitle}}</span>
+        <div class="title">{{label}}</div>
+        <div class="subtitle"> 
+          <span class="bastion" v-if="this.config.sshBastionHost">
+            <span class="truncate">{{ subtitleBastion }}</span>&nbsp;>&nbsp;
```

Comment:
no need for the computed properties, just use `this.config.sshBastionHost` here too